arbiter/server_test.go

```diff
@@ -407,7 +407,7 @@ func (s *syncBinlogsSuite) TestShouldSendBinlogToLoader(c *C) {
 	}()
 	ld := dummyLoader{input: dest}
 
-	err := syncBinlogs(source, &ld)
+	err := syncBinlogs(context.Background(), source, &ld)
```

Comment:
Add a test case here to show that `syncBinlogs` can be canceled from outside?

Reply:
I add another test case in the end